firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -69,8 +69,8 @@
    * <p>Note: Also initializes the Firebase installations SDK that creates installation IDs to
    * identify Firebase installations and periodically sends data to Firebase servers. Remote Config
    * requires installation IDs for Fetch requests. To stop the periodic sync, call {@link
-   * com.google.firebase.installations.FirebaseInstallations#delete()}. Sending a Fetch request after
-   * deletion will create a new installation ID for this Firebase installation and resume the
+   * com.google.firebase.installations.FirebaseInstallations#delete()}. Sending a Fetch request
+   * after deletion will create a new installation ID for this Firebase installation and resume the
```

Comment:
Fix the formatting issue in https://github.com/firebase/firebase-android-sdk/pull/1761#issuecomment-654470935.